src/main/java/edu/ucsb/mapache/entities/AppUser.java

```diff
@@ -89,9 +93,26 @@ public void setTime(long time){
   }
   @Override
   public String toString() {
-    return String.format("AppUser[ id=%d, email=%s, firstName=%s, lastName=%s, searchRemain=%d]", id, email, firstName, lastName, searchRemain);
+    return String.format("AppUser[ id=%d, email=%s, firstName=%s, lastName=%s, searchRemain=%d, customAPIToken=%s]", id, email, firstName, lastName, searchRemain, apiToken);
   }
 
+  public String getApiToken() {
+    return this.apiToken;
+  }
+
+  public void setApiToken(String apiToken) {
+    this.apiToken = apiToken;
+  }
+
+  public void clearApiToken() {
+    this.apiToken = "invalid token";
+  }
+
+  public AppUser apiToken(String apiToken) {
+    this.apiToken = apiToken;
+    return this;
+  }
```

Comment:
These two methods seem to duplicate the functionality of the getter and setter immediately above, is it really necessary to have these? Also, what is the second method `apiToken` used for?

Reply:
Yea, this is left-over code that the group before us had written and we didn't have a need to remove. We never use it for anything, but I remember Professor Conrad mentioning that it could have a use case (I don't remember what). I suppose we can delete it.

Reply:
We can always rewrite this code later, there's nothing complex going on in here so I think it's okay to remove?

Reply:
I agree in this case...   the code on lines 111-114 is an example of a "fluent setter" which is created by the Sohibe code generator.   We can get rid of it.  Or keep it.    I'm ok either way.

Reply:
In the interest of maintaining consistency with the rest of the entity and to not confuse future students, I think we should remove it, but if its removal causes problems then we should probably leave it in for now.